tpp.h

```diff
@@ -0,0 +1,42 @@
+/*******************************************************************************
+* Copyright 1999-2016 Intel Corporation All Rights Reserved.
```

Comment:
what's this? can we remove it? inserting this into a Apache software doesn't feel right